python_utils.py

```diff
@@ -418,6 +418,7 @@ def url_open(source_url):
     Returns:
         urlopen. The 'urlopen' object.
     """
+    # TODO(#12912): Remove pylint disable after the check is refactored.
```

Comment:
Which check is this referring to?

Reply:
`arg-name-for-non-keyword-arg`, updated the comment.